src/attribute/Factory.sol

```diff
@@ -13,8 +13,20 @@ import { Ownable } from "./Ownable.sol";
 /// @notice An abstract factory that manages creates and manages instances
 /// @dev Ownable and Pausable, and satisfies the IBeacon interface by default.
 abstract contract Factory is IFactory, Ownable, Pausable {
-    /// @notice The instances mapping storage slot
-    bytes32 private constant INSTANCE_MAP_SLOT = keccak256("equilibria.root.Factory.instances");
+    /// @custom:storage-location erc7201:equilibria.root.Factory
+    struct FactoryStorage {
+        mapping(IInstance instance => bool registered) instances;
+    }
+
+    /// @dev The erc7201 storage location of the mix-in
+    bytes32 private constant FactoryStorageLocation = 0x2068933510e31bb02be4765cf6d0b2c054190db3eddefe11ffed0ca32b7e6f00;
+
+    /// @dev The erc7201 storage of the mix-in
+    function Factory$() private pure returns (FactoryStorage storage $) {
```

Comment:
I haven't seen this naming standard before; will we be using `$` as a placeholder for `Storage`?  Linter traps that the function name is not mixedCase.  We usually prefix private method names with an underscore.  Is deviation from our normal style desirable here?

Reply:
Having reviewed the rest of the code, I like the naming; it makes it feel like this is a static method on the type.

Reply:
The common pattern is to do something like:
```
FactoryStorage storage $ = getFactoryStorage();
$.field = ...;
```

I originally thought to make this a little nicer by doing something like:
```
$().field = ...;
```

However solidity has an open issue where two inherited classes cannot have the same-named function even if both are private, so I had to prefix with the contract name like we see here.